src/graphdb.js

```diff
@@ -52,42 +52,81 @@ function toPascalCase (str) {
     return r.trim();
 }
 
+// Changes every instance of invalid characters in the given label with the following abbreviations
+function replaceCleanseLabel(label) {
+    return label
+        .replaceAll("!", "_ex_")
+        .replaceAll("$", "_dol_")
+        .replaceAll("&", "_amp_")
+        .replaceAll("(", "_op_")
+        .replaceAll(")", "_cp_")
+        .replaceAll(".", "_dot_")
+        .replaceAll(":", "_cn_")
+        .replaceAll("=", "_eq_")
+        .replaceAll("@", "_at_")
+        .replaceAll("[", "_os_")
+        .replaceAll("]", "_cs_")
+        .replaceAll("{", "_oc_")
+        .replaceAll("|", "_vb_")
+        .replaceAll("}", "_cc_")
+        .replaceAll("-", "_hy_");
+}
 
 function graphDBInferenceSchema (graphbSchema, addMutations) {
```

Comment:
Can you add some unit tests to `graphed.test.js` to ensure that special characters are escaped with back ticks? There is an existing test there which you can use as an example to provide input to the function and then verify the output.